tensorflow/contrib/cmake/tf_core_kernels.cmake

```diff
@@ -19,12 +19,14 @@ list(REMOVE_ITEM tf_core_kernels_srcs ${tf_core_kernels_exclude_srcs})
 
 add_library(tf_core_kernels OBJECT ${tf_core_kernels_srcs})
 
-add_dependencies(tf_core_kernels tf_core_cpu)
+add_dependencies(tf_core_kernels tf_core_cpu farmhash highwayhash)
```

Comment:
We also need dependencies on `farmhash_copy_headers_to_destination` and  `highwayhash_copy_headers_to_destination` here.